widget/Toaster.js

```diff
@@ -76,10 +78,11 @@ define([
 			this.hide();
 
 			// place node as a child of body for positioning
-			this.ownerDocument.body.appendChild(this.domNode);
+			this.domNode.ownerDocument.body.appendChild(this.domNode);
 
 			if(this.messageTopic){
-				connect.subscribe(this.messageTopic, this, "_handleMessage");
+				// TODO: wrap in a call to this.own
+				topic.subscribe(this.messageTopic, lang.hitch(this, "_handleMessage"));
```

Comment:
`this.subscribe()` is probably best, or second best `this.own(topic.subscribe(...))`.   And in either case removing the TODO comment.